src/main/resources/templates/ss.html

```diff
@@ -0,0 +1,78 @@
+                                <td style="padding:0 0 13px; font:16px/24px Arial;">Hi <a href="#"  style="text-decoration:none; color:#333; font-weight:bold;">XXX</a>,</td>
+                            </tr>
+                            <tr>
+                                <td style="padding:0 0 10px; font:16px/24px Arial;">Your account approval request has been<a href="#"  style="text-decoration:none; color:#333; font-weight:bold;"> APPROVED.</a><p>Your username is: <a href="#"  style="text-decoration:none; color:#333; font-weight:bold;">xxxxx@nus.edu.sg</a></p> </td>
+                            </tr>
+                            <tr>
+                                <td style="padding:0 0 11px; font:16px/24px Arial;"><p>Should you have any question, please don't hestitate to contact us. Thanks!<br /></p>
+                                </td>
+                            </tr>
+                            <tr>
+                                <td style="padding:0 0 18px; font:16px/24px Arial; color:#333;"><p>Best Regards,<br /> The NCLTeam <br /> Tel: (65) 6601 1056 <br/><br />
+                                </p>
+                                </td>
+                            </tr>
+                        </table>
+                    </td>
+                </tr>
+                <tr>
+                    <td bgcolor="#eeeeee" style="padding:20px 10px 20px 30px;">
+                        <table width="100%">
+                            <tr>
+                                <td>
+                                    <table width="138" align="left";>
+                                        <tr>
+                                            <td style="padding:8px 0 14px; font:16px/18px Arial, Helvetica, sans-serif; color:#999;">Connect with us</td>
+                                        </tr>
+                                    </table>
+                                    <table width="426" align="right">
+                                        <tr>
+                                            <td>
+                                                <table>
+                                                    <tr>
+                                                        <td width="14"></td>
+                                                        <td ><a href="mailto:support@ncl.sg" ><img src="email.png" border="0" style="vertical-align:top;" width="44" height="39"  /></a></td>
+                                                        <td width="14"></td>
+                                                        <td ><a href="https://github.com/nus-ncl" ><img src="github.png" border="0" style="vertical-align:top;" width="39" height="39"  /></a></td>
+                                                        <td width="14"></td>
+
+                                                        <td><a href="http://ncl.sg" ><img src="website.png" border="0" style="vertical-align:top;" width="39" height="39"  /></a></td>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add an "alt" attribute to this image. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)